src/components/PreviewFrame.jsx

```diff
@@ -139,6 +139,10 @@ class PreviewFrame extends React.Component {
     this.props.onConsoleLog(printedValue, compiledProjectKey);
   }
 
+  _handleSave() {
+    this.props.onSave();
+  }
```

Comment:
No big deal either way but I don’t see much advantage to having this in its own method vs. just calling the prop callback directly from the handler.

Reply:
Fair - it seemed a bit more consistent to do it this way, though I can see how its more verbose